scripts/import-sdvx.mjs

```diff
@@ -211,6 +213,7 @@ function buildSong(song, availableJackets) {
   return {
     name: info.title_name[0],
     search_hint: info.ascii[0],
+    distribution_date: info.distribution_date[0],
```

Comment:
I guess this would be actually in the songs array in ```songs.schema.json```, since it's in that same section for the other parts of the XML that is being pulled in, and it is not actually updated per chart, even when they add new difficulties to existing songs.

I have created the pull request for this, if you can please review, and update the variable name to ```date_added``` in this pull request.